src/MPDevice.cpp

```diff
@@ -5242,7 +5242,7 @@ bool MPDevice::readExportFile(const QByteArray &fileData, QString &errorString)
             else if  (encryptionMethod == "none")
             {
                 /* Legacy, not generated anymore */
-                return readExportPayload(QJsonDocument::fromJson(importFile.value("payload").toString().toUtf8()).array(), errorString);
+                return readExportPayload(QJsonDocument::fromJson(pMesProt->toByteArray(importFile.value("payload").toString())).array(), errorString);
```

Comment:
if I'm not mistaken, this is not needed here as you are reading a file generated by moolticute and not the device

Reply:
right, fixed it.